dev/CommandBarFlyout/CommandBarFlyout.idl

```diff
@@ -8,10 +8,7 @@ unsealed runtimeclass CommandBarFlyout : Windows.UI.Xaml.Controls.Primitives.Fly
 {
     CommandBarFlyout();
 
-    [MUX_PREVIEW]
-    {
-        Boolean AlwaysExpanded{ get; set; };
-    }
+    Boolean AlwaysExpanded{ get; set; };
```

Comment:
Since this is adding to an existing type, we need to version it V2. See  [https://github.com/microsoft/microsoft-ui-xaml/blob/b249e107a36d3a5302911762429c95c9c7106628/idl/Microsoft.UI.Xaml.idl#L110](https://github.com/microsoft/microsoft-ui-xaml/blob/b249e107a36d3a5302911762429c95c9c7106628/idl/Microsoft.UI.Xaml.idl#L110)